packages/loopring_v3/circuit/Circuits/TransferCircuit.h

```diff
@@ -450,11 +440,9 @@ class TransferCircuit : public BaseTransactionCircuit {
   const VariableArrayT getPublicData() const {
     return flattenReverse(
         {type.bits, fromAccountID.bits, toAccountID.bits, tokenID.bits,
-         feeTokenID.bits, fAmount.bits(), fFee.bits(),
-         nonce.getShortStorageID(), da_To.result(), da_ValidUntil.result(),
-         da_StorageID.result(), da_From.result(),
-         VariableArrayT(256 - NUM_BITS_FIELD_CAPACITY, state.constants._0),
-         data.bits});
+         fAmount.bits(), feeTokenID.bits, fFee.bits(),
```

Comment:
Why switch fAmount and feeTokenID sequence? If we switch here, we also need to modify TransferTransaction.sol to keep them consistent

Reply:
Forget about the comment, I see the TransferTransaction.sol is also modified.